src/createDIContainer.js

```diff
@@ -63,6 +65,13 @@ async function createDIContainer() {
     // `configCollection` and `config` are registering on command init
   });
 
+  /**
+   * Templates
+   */
+  container.register({
```

Comment:
You need to register `writeServiceConfigs` too. There are no dependencies so I would use `asValue` helper to register it.

Reply:
Why does it matter if a function has dependencies? It looks like `writeServiceConfigs` depends on `fs`, why should I use `asValue` there but not for `renderServiceTemplatesFactory`, which also depends only on external modules like `fs` and `dot`...?

Reply:
If you use `asFunction` helper, awilix looks for dependencies in the passed function arguments. It calls a factory. The factory creates a function with injected dependencies. You passing not a factory but a normal function you need to use `asValue` helper which means "as is".